src/Raygun.Samples.Blazor.WebAssembly/Raygun.Samples.Blazor.WebAssembly.csproj

```diff
@@ -5,6 +5,8 @@
     <ServiceWorkerAssetsManifest>service-worker-assets.js</ServiceWorkerAssetsManifest>
     <RootNamespace>Raygun.Samples.Blazor.WebAssembly</RootNamespace>
     <Nullable>enable</Nullable>
+    <!-- disable caching for testing purposes -->
+    <BlazorCacheBootResources>false</BlazorCacheBootResources>
```

Comment:
Yes, because this config file belongs to the sample app. If we don't disable cache, I had problems testing changes in code when running the sample project because the files were kept in the browser cache.